package.json

```diff
@@ -390,6 +390,11 @@
                     "default": "false",
                     "description": "Use an existing custom sysimage when starting the REPL",
                     "scope": "application"
+                },
+                "julia.forceDebug": {
```

Comment:
Would probably make sense to use a better name here that makes it clear that this is about the language server, right? This name here suggests to users that we have debugging support for julia in the extension, which we (unfortunately) don't.

Reply:
My English is not good so I have no suggestions on tip writing, thought I totally agree with you. A naive idea is to rename it to `julia.extension.debugMode`?